package.json

```diff
@@ -44,6 +44,7 @@
     "@patternfly/react-table": "4.111.33",
     "@rhoas/app-services-ui-shared": "^0.16.4",
     "ajv": "^8.8.2",
+    "dagre": "^0.8.5",
```

Comment:
If I'm not mistaken, it is this package which is used https://www.npmjs.com/package/dagre which is marked as deprecated https://github.com/dagrejs/dagre with last release which is 3 years old.

Do you feel confident that there would be a new release if needed?

Reply:
@apupier How about we merge what we have for now so that we can get feedback on branching and autolayout, and I can swap the layouting implementation underneath into next week? So as not to block this. Or do you think it'd be best to keep open until making this change?

Reply:
Given that it is an highly awaited feature, I think we can go to merge with this dependency and create an issue with Critical(?) state to move to another implementation of layouting.